onnxruntime/core/providers/nv_tensorrt_rtx/onnx_ctx_model_helper.h

```diff
@@ -24,6 +24,11 @@ static const std::string PARTITION_NAME = "partition_name";
 static const std::string SDK_VERSION = "ep_sdk_version";
 static const std::string EPCONTEXT_OP_DOMAIN = "com.microsoft";
 
+constexpr size_t kTensorRTEngineHeaderSize = 64;
```

Comment:
Would it be possible to have this define in a TRT header ? This seems a bit risky to maintain if things change.